.browserslistrc

```diff
@@ -0,0 +1,18 @@
+[production]
+>0.5% in alt-eu and >0.5% in alt-as and >0.5% in alt-na and >0.5% in alt-sa and >0.5% in alt-af and >0.5% in alt-oc and >1%
+not ie > 0
+last 2 Chrome versions
+last 2 Edge versions
+last 2 Safari versions
+Firefox esr
```

Comment:
aren't we missing `last 2 Firefox versions` in this list?

Reply:
does `Firefox esr` already imply only the latest ESR?

Reply:
Firefox ESR is always lower than the last 2 Firefox versions, so, it's already the lowest version possible...

Reply:
> Firefox ESR: the latest [Firefox ESR] version.
(from [here](https://github.com/browserslist/browserslist))